mungegithub/mungers/sync/issue-sync.go

```diff
@@ -32,14 +32,9 @@ const (
 	// JenkinsBotName is the name of kubekins bot
 	JenkinsBotName = "k8s-bot"
 	priorityPrefix = "priority/P"
-	// PriorityP0 represents Priority P0
-	PriorityP0 = Priority(0)
-	// PriorityP1 represents Priority P1
-	PriorityP1 = Priority(1)
-	// PriorityP2 represents Priority P2
-	PriorityP2 = Priority(2)
-	// PriorityP3 represents Priority P3
-	PriorityP3 = Priority(3)
+
+	// PriorityFailingTest represents a failing or flaking test
+	PriorityFailingTest = Priority(2)
 )
```

Comment:
I don't think the interface would need to be changed.  The function `Priority` still returns `(Priority, bool)`.  The return is now hardcoded for test runs and flakes, but this framework allows us to easily alter prioritization in the future.